api/v1/authenticator_types.go

```diff
@@ -150,6 +153,20 @@ type Ports struct {
 type AuthenticatorStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// Interfaces is the list of interface status
+	// +optional
+	Interfaces []*Interface `json:"interfaces,omitempty"`
+}
+
+type Interface struct {
+	// Name is the name of the interface
+	Name string `json:"name"`
+	// State is the state of the interface
```

Comment:
If these state values are from somewhere (from hostapd itself?) we should mention what the expected values are.

Reply:
yes, done.